internal/providers/terraform/testdata/hcl_provider_test/populates_warnings_on_missing_vars/expected.json

```diff
@@ -0,0 +1,74 @@
+{
```

Comment:
I was expecting to see a warning in this output file, but it doesn't look like there is one.

Reply:
This is plan JSON output (which the hcl parser builds to pass to the json parser) so the warnings don't live in here. I'm testing they come through on the parser here: internal/providers/terraform/hcl_provider_test.go. But I'll also add a breakdown integration test so that we have coverage.

Reply:
Ah okay, that makes sense. I think I was thrown off by the `infracost_metadata` we add in here.

Reply:
added `breakdown` format JSON test in https://github.com/infracost/infracost/pull/2037/commits/020801f203d427bb8be8d2f9de436aaee5420426